src/routes/docs/products/auth/mfa/+page.markdoc

```diff
@@ -0,0 +1,1018 @@
+Future result = account.createMfaChallenge(
+factor: 'email',
+);
+
+result.then((response) {
+    print(response);
+    // Save the challenge ID to complete the challenge later
+    var challengeId = response.$id;
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let mfaChallenge = try await account.createMfaChallenge(
+  factor: "email"
+)
+// Save the challenge ID to complete the challenge later
+let challengeId = mfaChallenge.id
+```
+
+```client-android-kotlin
+val response = account.createMfaChallenge(
+    factor = "email"
+)
+// Save the challenge ID to complete the challenge later
+val challengeId = response.id
+```
+{% /multicode %}
+{% /tabsitem %}
+
+{% tabsitem #phone title="Phone" %}
+Appwrite will use a verified phone number on the user's account to send the challenge code via SMS.
+You cannot use this method if the user initialized their login with phone OTP.
+
+{% multicode %}
+```client-web
+const challenge = await account.createMfaChallenge(
+        'phone'                       // factor
+    );
+
+// Save the challenge ID to complete the challenge later
+const challengeId = challenge.$id;
+```
+
+```client-flutter
+Future result = account.createMfaChallenge(
+factor: 'phone',
+);
+
+result.then((response) {
+    print(response);
+    // Save the challenge ID to complete the challenge later
+    var challengeId = response.$id;
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let mfaChallenge = try await account.createMfaChallenge(
+  factor: "phone"
+)
+// Save the challenge ID to complete the challenge later
+let challengeId = mfaChallenge.id
+```
+
+```client-android-kotlin
+val response = account.createMfaChallenge(
+    factor = "phone"
+)
+// Save the challenge ID to complete the challenge later
+val challengeId = response.id
+```
+{% /multicode %}
+{% /tabsitem %}
+
+{% tabsitem #totp title="TOTP" %}
+
+Initiate a challenge for users to complete using an authenticator app.
+
+{% multicode %}
+```client-web
+const challenge = await account.createMfaChallenge(
+        'totp'                       // factor
+    );
+
+// Save the challenge ID to complete the challenge later
+const challengeId = challenge.$id;
+```
+
+```client-flutter
+Future result = account.createMfaChallenge(
+factor: 'totp',
+);
+
+result.then((response) {
+    print(response);
+    // Save the challenge ID to complete the challenge later
+    var challengeId = response.$id;
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let mfaChallenge = try await account.createMfaChallenge(
+  factor: "totp"
+)
+// Save the challenge ID to complete the challenge later
+let challengeId = mfaChallenge.id
+```
+
+```client-android-kotlin
+val response = account.createMfaChallenge(
+    factor = "totp"
+)
+// Save the challenge ID to complete the challenge later
+val challengeId = response.id
+```
+{% /multicode %}
+{% /tabsitem %}
+{% /tabs %}
+{% /section %}
```

Comment:
We are missing a recovery code factor examples

Reply:
Done